test/integration_test/extender_test.go

```diff
@@ -29,7 +30,10 @@ func TestExtender(t *testing.T) {
 
 func noPVCTest(t *testing.T) {
 	ctxs, err := schedulerDriver.Schedule(generateInstanceID(t, "nopvctest"),
-		scheduler.ScheduleOptions{AppKeys: []string{"mysql-nopvc"}})
+		scheduler.ScheduleOptions{
+			AppKeys:   []string{"mysql-nopvc"},
+			ConfigMap: authTokenConfigMap,
```

Comment:
Why does this need to be passed in everywhere? We should just initialize the schedulerDriver with it and let it add it to the required specs. Requiring to pass this in everywhere is error-prone.

Reply:
This will be fixed in this PR:  https://github.com/portworx/torpedo/pull/368

Reply:
This has been completed.